lib/engine/game/g_18_usa/step/acquire.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
```

Comment:
Revert

Reply:
I just force pushed the latest set of changes. I don't think that for the acquisition round (probably for merger but I need to look at that before I can say it with the say confidence) that reverting this and adding a scrap_train step makes sense.

In the 1817 Acquisition round, it is implemented mainly in the Acquisition step as  a series of possible actions with a few other steps for interrupting steps (bankruptcy, cash crisis, forced train discard, forced token reduction) so I believe that the best way here is to insert the ability to scrap a pullman when a corporation is up to act in the round (as opposed to a player)